### PR TITLE
Added link to latest release

### PR DESCRIPTION
**Pull request by duncanpMS**
Updated the SDK ReadMe to contain links to the release zip, and to contain brief instructions on how to embed SQALE information.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,7 +19,7 @@ There are two groups of target users:
    - If the analyzer author has not provided a SonarQube plugin for their analyzer then users will be able to generate a plugin from an analyzer NuGet package, although they won't be able to provide such rich metadata.
 
 ### Current status
-The generated jar file is not currently sufficient on its own to upload Roslyn issues. The complete solution requires the [C# plugin v4.5](http://docs.sonarqube.org/display/PLUG/C%23+Plugin) or higher and the [MSBuild SonarQube Scanner v2.0](http://docs.sonarqube.org/display/SONAR/Analyzing+with+SonarQube+Scanner+for+MSBuild) or higher.
+The generated jar file is not  sufficient on its own to upload Roslyn issues. The complete solution requires the [C# plugin v4.5](http://docs.sonarqube.org/display/PLUG/C%23+Plugin) or higher and the [MSBuild SonarQube Scanner v2.0](http://docs.sonarqube.org/display/SONAR/Analyzing+with+SonarQube+Scanner+for+MSBuild) or higher. See [this](https://blogs.msdn.microsoft.com/visualstudioalm/2016/02/18/sonarqube-scanner-for-msbuild-v2-0-released-support-for-third-party-roslyn-analyzers/) blog post for more information on how the various components cooperate to handle issues from Roslyn analyzers.
```
Comment (dbolkensteyn) [inline README.md]: @duncanpMS imo this sounds a bit strange `Current status: The generated jar file is not  sufficient on its own to upload Roslyn issues.`

My first impression when I read just that is that this thing isn't ready to be used yet.

Comment (dbolkensteyn) [inline README.md]: it's better to rely on tinylinks on the documentation, in cases these pages are renamed or moved.

For example the C# plugin page's link is http://docs.sonarqube.org/x/bAAW

To get it, click on "..." in Confluence and then "Link to this page"

**Comment by dbolkensteyn** [comment]
LGTM

File: README.md (hunk 1 above)
Comment (duncanpMS) [inline README.md]: Thanks @dbolkensteyn. I'll reword the "Current status" text to something like "The generated jar works with the C# plugin and SonarQube Scanner for MSBuild to provide support for configuring rules and uploading issues from the Roslyn analyser".
I'll change the links too.

**Pull request merged**